deepchecks/tabular/utils/task_inference.py

```diff
@@ -67,22 +65,18 @@ def get_possible_classes(model: Optional[BasicModel], train_dataset: 'tabular.Da
             get_logger().warning('Model classes attribute does not contain all observed labels in train and test data.')
         observed_labels += list(model.classes_)
 
-    if model is not None and not hasattr(model, 'predict_proba') and not force_classification:  # regression model
-        if is_string_column(train_dataset.label_col):
-            reference = doclink('supported-prediction-format', template='For additional details see {link}')
-            raise DeepchecksValueError(f'Classification models must contain \'predict_proba\' functionality. '
-                                       f'{reference}')
-        if isinstance(model, ClassifierMixin):
+    if model is not None:  # classification model without classes_ attribute
```

Comment:
Not sure it is required, why do we care? only raise exception if probas are required by scorers